tests/Doctrine/Tests/ORM/Hydration/ObjectHydratorTest.php

```diff
@@ -927,10 +928,10 @@ public function testEntityQueryCustomResultSetOrder(): void
         self::assertEquals(1, $result[0]->getId());
         self::assertEquals(2, $result[1]->getId());
 
-        self::assertObjectHasAttribute('boards', $result[0]);
+        self::assertTrue(property_exists($result[0], 'boards'));
```

Comment:
`assertObjectHasAttribute()` has been removed without replacement. 😢

Reply:
That would be because dynamic properties are deprecated I suppose :thinking:

Reply:
Yes, and because the name `assertObjectHasAttribute()` is somewhat confusing with PHP 8.